src/pages/monetization/app-review-process/index.mdx

```diff
@@ -26,37 +26,74 @@ During this process, we will make sure that:
 - The app matches the LiveChat's look and feel. For that reason, we highly recommend following our Design System.
 - The app has all the necessary Marketplace listing details: app icon, images, description, key features and benefits, a tutorial, and privacy policy.
 
-# Best practices
+## App quality requirements
+
+There are several key requirements that determine whether your application will successfully pass the app review. These guidelines ensure that your app meets the necessary standards for functionality, user experience, and overall quality. All developers must adhere to these requirements in order for their apps to be able to publish their apps on the Marketplace.
+
+### Design System
+
+We require using our Design System to keep apps consistent with the LiveChat look & feel.
+
+Read more about apps’ components in the <a href="https://design.livechat.com/" target="_blank" rel="noopener noreferrer">**Design System**</a>, and to learn more about colors and typography, please go <a href="https://www.figma.com/file/XmaxDzzLwCVXrMYQyjvyx6/Light-Theme-Tokens?type=design&node-id=0%3A1&t=Jy10v7JrSZCviFwS-1" target="_blank" rel="noopener noreferrer">**here**</a>.
```

Comment:
Access to Figma is on request - not cool. Overall, this whole Figma will definitely be overwhelming. I doubt developers will analyze this much data and use those tokens correctly. TBH, I think we should create a simplified version of this

Reply:
I'll change this link to our Storybook